src/models/users-schema.js

```diff
@@ -52,16 +80,16 @@ users.statics.authenticateBasic = async function(auth) {
  * Because this is a methods function, `this` refers to an individual user record
  * @return {string} The generated jwt token
  */
-// === TODO implement timeout functionality for this token ====
-// === You can have your code pass generateToken a flag that ===
-// === sets a long or short (5 sec) timeout ===
-users.methods.generateToken = function() {
+
+users.methods.generateToken = function(timeout) {
+  let exp = Math.floor(Date.now() / 1000 + 60 * 60);
```

Comment:
You may need to check if `timeout === true` before you set it to one of the Math functions